src/pages/spawn/spawnHost/SpawnHostModal.tsx

```diff
@@ -73,6 +80,25 @@ export const SpawnHostModal: React.FC<SpawnHostModalProps> = ({
     MyHostsQueryVariables
   >(GET_MY_VOLUMES);
 
+  // UPDATE HOST STATUS MUTATION
+  const [spawnHostMutation, { loading: loadingSpawnHost }] = useMutation<
+    SpawnHostMutation,
+    SpawnHostMutationVariables
+  >(SPAWN_HOST, {
+    onCompleted(hostMutation: SpawnHostMutation) {
```

Comment:
you don't need to include the type here--it should already be strongly typed from lines 86 and 87

Reply:
nice catch didnt think it was